tests/phpMyFAQ/Instance/ClientTest.php

```
@@ -93,4 +106,370 @@ public function testDeleteClientFolder(): void
+        );
+
+        Database::factory('pdo_pgsql');
+        Database::setTablePrefix('');
+
+        $dbMock = $this->createMock(DatabaseDriver::class);
+        $loggerMock = $this->createMock(Logger::class);
+
+        $this->configuration->method('getDb')->willReturn($dbMock);
+        $this->configuration->method('getLogger')->willReturn($loggerMock);
+
+        $queryCount = 0;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$queryCount'. (undefined)

(UnusedLocalVariable)